src/components/ItemStatus/ItemStatus.container.ts

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux'
+
+import { RootState } from 'modules/common/types'
+import { MapStateProps, MapDispatchProps, MapDispatch, OwnProps } from './ItemStatus.types'
+import { getStatusByItemId } from 'modules/item/selectors'
+import ItemStatus from './ItemStatus'
+
+const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
+  const statusByItemId = getStatusByItemId(state)
+  return {
+    status: statusByItemId[ownProps.item.id]
+  }
+}
+
+const mapDispatch = (_dispatch: MapDispatch): MapDispatchProps => ({})
```

Comment:
The convention for unused variables is just an undescore `_`, no need to declare the name (undescore and name is the convention for private)

Reply:
mm afaik typescript allows `_` if you don't use any variable but also `_param` if you need to select them individually, ie:

```ts
function (_paramA: any, _paramB: any, paramC: any) {
  // use only paramC
}
```

I think the `_param` is clearer and i think it's been used like that throughout the codebase